dialog/fileitem.go

```diff
@@ -57,6 +57,9 @@ func fileParts(path string) (name, ext string) {
 	if len(ext) > 1 {
 		ext = ext[1:]
 	}
+	if len(ext) > 10 {
```

Comment:
From your screenshots it looks unlikely that more than 5 characters will fit... it’s probably better to have it sometimes truncates too much (iiiii) than still overlap at times (mmmmmmmmmm). I’d also consider defining a local const or naming a variable for this number.

Reply:
Yeah okay. 5 is another arbitrary number and may not work for all resolutions but you're right that it is better to over truncate than spill over into other elements. I can't think of a better solution

Reply:
I agree it is arbitrary, but not sure what you mean about not working on some resolutions? This is all scaled proportionately so the only thing that would change the proportions is a custom theme that changes font size or icon size.

Reply:
Yup, I was thinking that really small screens, or really large screens could surface the limitations of this approach, but you're right that it _should_ all get scaled properly. Custom themes pose more of an issue, but I don't think we can protect against all customizations. This arbitrary number fixes _most_ issues, so let's go with it for now, and we can always improve it later